scripts/inventory/getinventory.py

```diff
@@ -48,7 +48,7 @@ def import_read_inventory_file():
 def get_instance_group(instance, group):
     read_inventory_file = import_read_inventory_file()
     servers = read_inventory_file(
-        os.path.join(ROOT, 'fab', 'fab', 'inventory', instance))
+        os.path.join(ROOT, 'environments', instance, 'inventory.ini'))
```

Comment:
Ah, I was wondering if this would be addressed since I ran into another issue with it today. I noticed that you must be in (have cwd set to) the root of the repo to run `./scripts/inventory/ssh`. I was in the `./ansible` subdir and tried
```
(ansible)::ansible$ ../scripts/inventory/ssh softlayer ansible@riakcs:0
python: can't open file 'scripts/inventory/getinventory.py': [Errno 2] No such file or directory
```
Would be really cool if we could `commcare-cloud ssh softlayer ansible@riakcs:0`  from anywhere (assuming the virtualenv is activated).

Reply:
Yeah that's occurred to me too, could be pretty neat. The one thing I'm not sure about is that `commcare-cloud` is a python script and python scripts aren't that great at handing over control to whatever process they're forking (or maybe I just do this wrong)—so I'm not sure how well dumping the user in an interactive shell works. Do you know what I mean here, and do you know how to get around that?

Reply:
Another lower-touch way to fix this is to use `$(dirname $0)` to get the script's directory. PR'd that over here: https://github.com/dimagi/commcarehq-ansible/pull/1218